enh(hardware-storage-hp-msa2000-snmp) CTOR-1985 (#4769)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/hardware-storage-hp-msa2000-snmp.md

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

## Connector dependencies

The following monitoring connectors will be installed when you install the **HP MSA2000 SNMP** connector through the

> **Configuration > Monitoring Connector Manager** menu:

Reviewer: ```suggestion
**Configuration > Connectors > Monitoring Connectors** menu:
```

</TabItem>
<TabItem value="Not attached to a host template" label="Not attached to a host template">

| Service Alias        | Service Template                                  | Service Description                          | Discovery |
|:---------------------|:--------------------------------------------------|:---------------------------------------------|:---------:|
| Traffic-Generic-Id   | HW-Storage-Hp-Msa2000-Traffic-Generic-Id-custom   | Check traffic of an network interface        |           |

> | Traffic-Generic-Name | HW-Storage-Hp-Msa2000-Traffic-Generic-Name-custom | Check traffic of an network interface        |           |

Reviewer: ```suggestion
| Traffic-Generic-Id   | HW-Storage-Hp-Msa2000-Traffic-Generic-Id-custom   | Check traffic of a network interface        |           |
| Traffic-Generic-Name | HW-Storage-Hp-Msa2000-Traffic-Generic-Name-custom | Check traffic of a network interface        |           |
```

### Pack

1. If the platform uses an *online* license, you can skip the package installation
instruction below as it is not required to have the connector displayed within the

> **Configuration > Monitoring Connector Manager** menu.

Reviewer: ```suggestion
**Configuration > Connectors > Monitoring Connectors** menu.
```

</TabItem>
</Tabs>

2. Whatever the license type (*online* or *offline*), install the **HP MSA2000 SNMP** connector through

> the **Configuration > Monitoring Connector Manager** menu.

Reviewer: ```suggestion
the **Configuration > Connectors > Monitoring Connectors** menu.
```
